blockchain_connector/avalon_blockchain_connector/fabric/fabric_connector.py

```diff
@@ -75,17 +76,43 @@ def sync_worker(self):
         """
         Check for existing worker and update worker to fabric blockchain
         """
-        worker_type = WorkerType.TEE_SGX
+        # First get all the worker from blockchain
```

Comment:
Change "all the worker from" to "all the workers from the"

Reply:
Done